controls/KEYC-01-000006.rb

```diff
@@ -35,4 +52,16 @@
   tag stig_id: "KEYC-01-000006"
   tag cci: ["CCI-001403"]
   tag nist: ["AC-2 (4)"]
+
+  program = "/opt/keycloak/bin/kcadm.sh get events/config -r #{input('keycloak_realm')}"
+
+  describe json(content: command(program).stdout) do
+	  its('eventsEnabled') { should eq input('events_enabled') }
+	  its('eventsListeners') { should cmp input('events_listeners') }
+	  its('enabledEventTypes') { should include input(['enabled_event_types', 'value'], value: "UPDATE_PROFILE") }
+	  its('enabledEventTypes') { should include input(['enabled_event_types', 'value'], value: "UPDATE_EMAIL") }
+	  its('enabledEventTypes') { should include input(['enabled_event_types', 'value'], value: "UPDATE_PASSWORD") }
+	  its('adminEventsEnabled') { should eq input('admin_events_enabled') }
```

Comment:
Same comment about this one. Should it be an input? It has to be true to meet the spec.